From 09c49c6d6eee6820797e368e8e419d7f191de726 Mon Sep 17 00:00:00 2001
From: Matthew McLeod <mtthwmcld@gmail.com>
Date: Mon, 20 Dec 2021 19:27:32 +0100
Subject: chore: remove README-helm3.md (#36561)

* chore: Update README with gateways in seperate ns

* chore: remove README-helm3.md

Out of date, please see https://istio.io/latest/docs/setup/install/helm/.
---
 manifests/charts/README-helm3.md | 66 --------------------------------
 1 file changed, 66 deletions(-)
 delete mode 100644 manifests/charts/README-helm3.md

diff --git a/manifests/charts/README-helm3.md b/manifests/charts/README-helm3.md
deleted file mode 100644
index ad33617929..0000000000
--- a/manifests/charts/README-helm3.md
+++ /dev/null
@@ -1,66 +0,0 @@
-# Helm v3 support
-
-## Install
-
-The Helm charts are supported both by Helm v2 and Helm v3. Please do not introduce Helm v3 specific changes as many
-users are still using Helm v2 and the operator is currently using the Helm v2 code to generate.
-
-To install with Helm v3, you must first create the namespace that you wish to install in if the namespace does not exist already. The default namespace used is `istio-system` and can be created as follows:
-
-```console
-kubectl create namespace istio-system
-```
-
-The charts are as follows:
-
-- `base` creates cluster-wide CRDs, cluster bindings and cluster resources. It is possible to change the namespace from `istio-system` but it is not recommended.
-
-```console
-helm install istio-base -n istio-system manifests/charts/base
-```
-
-- `istio-control/istio-discovery` installs a revision of istiod.  You can install it multiple times, with different revisions.
-
-```console
- helm install -n istio-system istio-17 manifests/charts/istio-control/istio-discovery
-
- helm install -n istio-system istio-canary manifests/charts/istio-control/istio-discovery \
-    --set revision=canary
-
- helm install -n istio-system istio-mytest manifests/charts/istio-control/istio-discovery \
-    --set revision=mytest
-```
-
-- `gateways` install a load balancer with `ingress` and `egress`. You can install it multiple times with different revisions but they must be installed in separate namespaces.
-
-Ingress secrets and access should be separated from the control plane.
-
-```console
-helm install -n istio-system istio-ingress manifests/charts/gateways/istio-ingress
-
-kubectl create ns istio-ingress-canary
-helm install -n istio-ingress-canary istio-ingress-canary manifests/charts/gateways/istio-ingress \
-    --set revision=canary
-```
-
-Egress secrets and access should be separated from the control plane.
-
-```console
-helm install -n istio-system istio-egress manifests/charts/gateways/istio-egress
-
-kubectl create ns istio-egress-canary
-helm install -n istio-egress-canary istio-egress-canary manifests/charts/gateways/istio-egress \
-    --set revision=canary
-```
-
-This is an optional step. [More details](install-OpenShift.md)
-
-- `istio-cni` installs the CNI plugin. This should be installed after the `base` chart and prior to `istiod`. Need to add `--set istio_cni.enabled=true` to the `istiod` install to enable its usage.
-
-```console
-helm install istio-cni -n kube-system manifests/charts/istio-cni
-```
-
-## Namespaces
-
-One of the changes in Helm v3 is that the namespace is no longer created on the fly when installing a chart. This means that the namespace being used needs to be created prior to installing the charts if it does not exist already. If the default `istio-system` namespace is not being used then you need to add the setting `--set global.istioNamespace=<namespace>` to the installs, to match the control plane namespace.
-- 
2.35.3

